### PR TITLE
Add isTranslatedIn method to TranslationObject class

### PR DESCRIPTION
**Pull request by void0-0**
title says it all ;)

### CODE REVIEW

File: packages/core/models/translation-object.model.ts [hunk 1]
```diff
@@ -41,6 +41,11 @@ export class TranslationObject {
         return { ...values };
     }
 
+    public isTranslatedIn(languages: string[]): boolean {
+        const translatedLanguages = Object.keys(this.getAll());
+        return languages.every(language => translatedLanguages.includes(language));
+    }
```
Comment (JulienDuf) [inline packages/core/models/translation-object.model.ts:47]: I think that `every` of an empty array will return true, wich is not the expected behaviour. You should test that edge case

Comment (void0-0) [reply packages/core/models/translation-object.model.ts:47]: Indeed

**Review by JulienDuf — APPROVED** [review]
